libs/sdk-backend-spi/src/workspace/execution/index.ts

```diff
@@ -151,6 +152,12 @@ export interface IPreparedExecution {
      */
     execute(): Promise<IExecutionResult>;
 
+    /**
+     * Triggers download of debugging files. If explain type is not specified, all files are included
+     * into a zip file.
+     */
+    withDebugAfm(debugConfig?: IDebugConfig): IPreparedExecution;
```

Comment:
I'm not sure about this, this means that bear backends have to support this too. Could we somehow use the tigerSpecificFucntions instead?